usagov/usagov-2021
# USAGOV-1995 new scam tool w3 c markup validation errors

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1995

## Description

This change resolves HTML validation errors caught by running this:
https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.usa.gov%2Fwhere-report-scams%2Fwhere-did-scam-take-place
This change also cleans up the sidenav code and makes it more reusable so if we decide to create custom sidenavs in the future, it will be simple to use the existing template rather than creating new templates that try to generate the same sidenav structure on their own.


## Testing Instructions


- Visit localhost/where-report-scams/where-did-scam-take-place
- View Page Source (Dont use dev tools to inspect since it automatically corrects some invalid HTML)
- Copy all the HTML
- Visit https://validator.w3.org/#validate_by_input
- Paste the copied HTML into the textarea
- Click the Check button
- Confirm that there are no errors or warnings beyond the following:
  - Bad values for `ID` attributes on Labels and `for` attributes on Inputs.
    - These have been fixed in content on Prod
  - Duplicate attribute class.
    - This is being resolved by another ticket

## Type of Changes

  - Add screenshot showing what it should look like

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
